src/main/java/nl/rijksoverheid/mev/gezagsmodule/domain/gezagvraag/OuderOverledenOfOnbevoegdTotGezag.java

```diff
@@ -28,36 +42,43 @@ protected OuderOverledenOfOnbevoegdTotGezag(final GezagsBepaling gezagsBepaling)
     @Override
     public void perform() {
         Persoonslijst plPersoon = gezagsBepaling.getPlPersoon();
-        answer = V4A_3_JA;
-        
-        // Preconditie: minimaal 1 ouder moet een BSN hebben
-        Ouder1 persoonOuder1 = plPersoon.getOuder1();
-        Ouder2 persoonOuder2 = plPersoon.getOuder2();
-        if ((persoonOuder1 == null || persoonOuder1.getBurgerservicenummer() == null)
-            && (persoonOuder2 == null || persoonOuder2.getBurgerservicenummer() == null)) {
+        var heeftOuder1Burgerservicenummer = plPersoon.getOuder1AsOptional().map(Ouder1::getBurgerservicenummer).isPresent();
+        var heeftOuder2Burgerservicenummer = plPersoon.getOuder2AsOptional().map(Ouder2::getBurgerservicenummer).isPresent();
+        if (!heeftOuder1Burgerservicenummer && !heeftOuder2Burgerservicenummer) {
             throw new AfleidingsregelException("Preconditie: Ouder moet een BSN hebben", "Ouder moet een BSN hebben");
         }
-        Persoonslijst lplOuder1 = gezagsBepaling.getPlOuder1();
-        Persoonslijst lplOuder2 = gezagsBepaling.getPlOuder2();
-        if ((lplOuder1 == null) && (lplOuder2 == null)) {
+
+        Persoonslijst persoonslijstOuder1 = gezagsBepaling.getPlOuder1();
+        Persoonslijst persoonslijstOuder2 = gezagsBepaling.getPlOuder2();
+        if ((persoonslijstOuder1 == null) && (persoonslijstOuder2 == null)) {
```

Comment:
sonar, de () om de conditie kunnen weg

Reply:
Weet je zeker het een Sonar issue is? Sonar geeft bij mij 0 issues. Onze SonarQube Cloud integratie geeft ook geen nieuwe issues aan.

Los daarvan ben ik het er mee eens dat onnodige brackets verwijderd kunnen worden.

```suggestion
        if (persoonslijstOuder1 == null && persoonslijstOuder2 == null) {
```